src/main/java/uk/gov/hmcts/divorce/caseworker/event/page/CorrectPaperCase.java

```diff
@@ -0,0 +1,195 @@
+                .mandatory(Application::getApplicant2SolStatementOfReconciliationFirm, "applicationType=\"jointApplication\"")
+                .label("Label-CorrectJurisdictionDetails", "### Jurisdiction connection details")
+                .complex(Application::getJurisdiction)
+                    .mandatory(Jurisdiction::getConnections)
+                .done()
+                .label("Label-CorrectPrayerDetails", "### Prayer details")
+                .readonly(Application::getDateSubmitted)
+                .mandatory(Application::getApplicant1ScreenHasMarriageBroken)
+                .mandatory(Application::getApplicant2ScreenHasMarriageBroken)
+                .mandatory(Application::getApplicant1PrayerHasBeenGivenCheckbox)
+                .mandatory(Application::getApplicant2PrayerHasBeenGivenCheckbox)
+                .label("Label-CorrectApp1HWFDetails",
+                    "### ${labelContentApplicantsOrApplicant1s} Help With Fees details")
+                .complex(Application::getApplicant1HelpWithFees)
+                    .optional(HelpWithFees::getReferenceNumber)
+                    .mandatory(HelpWithFees::getAppliedForFees)
+                    .mandatory(HelpWithFees::getNeedHelp)
+                .done()
+                .label("Label-CorrectApp2HWFDetails",
+                    "### ${labelContentRespondentsOrApplicant2s} Help With Fees details")
+                .complex(Application::getApplicant2HelpWithFees)
+                    .optional(HelpWithFees::getReferenceNumber)
+                    .mandatory(HelpWithFees::getAppliedForFees)
+                    .mandatory(HelpWithFees::getNeedHelp)
+                .done()
+                .mandatory(Application::getScreenHasMarriageCert)
+                .label("Label-CorrectMarriageDetails", "### Marriage details")
+                .complex(Application::getMarriageDetails)
+                    .mandatory(MarriageDetails::getCertificateInEnglish)
+                    .mandatory(MarriageDetails::getCertifiedTranslation,
+                        "marriageCertificateInEnglish=\"No\"")
+                    .mandatory(MarriageDetails::getMarriedInUk)
+                    .mandatory(MarriageDetails::getIssueApplicationWithoutMarriageCertificate)
+                    .mandatory(MarriageDetails::getDate)
+                    .mandatory(MarriageDetails::getApplicant1Name)
+                    .mandatory(MarriageDetails::getApplicant2Name)
+                    .mandatory(MarriageDetails::getFormationType)
+                    .mandatory(MarriageDetails::getPlaceOfMarriage)
+                    .mandatory(MarriageDetails::getCertifyMarriageCertificateIsCorrect)
+                    .mandatory(MarriageDetails::getMarriageCertificateIsIncorrectDetails,
+                        "marriageCertifyMarriageCertificateIsCorrect=\"No\"")
+                .done()
+            .done();
+    }
+
+    public AboutToStartOrSubmitResponse<CaseData, State> midEvent(
```

Comment:
are we missing configuring MidEvent on page ?

Reply:
Yes, you're right. Well spotted 🙏